### PR TITLE
Add example of importing a single SCSS file

### PR DESCRIPTION
**Pull request by danoc**

### CODE REVIEW

File: packages/thumbprint-scss/README.md [hunk 1]
```diff
@@ -42,6 +42,12 @@ For projects that want to compile Thumbprint CSS components along with other SCS
 @import '[node_modules path]/@thumbtack/thumbprint-scss/components
 ```
 
+You can import a single component (`button.css`, for example) with:
+
+```scss
+@import '[node_modules path]/@thumbtack/thumbprint-scss/button
```
Comment (webbower) [inline packages/thumbprint-scss/README.md]: Is `[node_modules path]/` necessary? I see a lot of uses without it that just start with `@thumbtack`

Comment (tomgenoni) [reply packages/thumbprint-scss/README.md]: It depends on if the project it's being used in is setup to resolve to `node_modules`. https://github.com/sass/node-sass/issues/2445

Comment (danoc) [reply packages/thumbprint-scss/README.md]: I've added more info on this.

**Comment by danoc** [comment]
This is ready for another look.

**Pull request merged**